frontend/__tests__/services/user-service.server.test.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpResponse } from 'msw';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+
+import { getUserService } from '~/services/user-service.server';
+
+global.fetch = vi.fn();
```

Comment:
I think you should mock globals using `vi.stubGlobal(..)`. I'm not 100% sure, but I think that directly overwriting `global.fetch` could have unindented side effects in unrelated tests.

<https://vitest.dev/guide/mocking.html#globals>

Reply:
Changed to subglobal

Reply:
For some reason, github wouldn't let me add to this comment, so see <https://github.com/DTS-STN/canadian-dental-care-plan/pull/1758#discussion_r1646034919>